README.md

```diff
@@ -1,185 +1,164 @@
-# Installing FUN CMS
+# Richie, A FUN CMS for Open edX
 
 [![Greenkeeper badge](https://badges.greenkeeper.io/openfun/richie.svg)](https://greenkeeper.io/)
 
 [![CircleCI](https://circleci.com/gh/openfun/richie/tree/master.svg?style=svg)](https://circleci.com/gh/openfun/richie/tree/master)
 
-This document aims to list all needed steps to have a working `FUN CMS` installation on your laptop.
+## Overview
 
-A better approach is to use [`Docker`](https://docs.docker.com). To get started, you can read [explanations](docs/docker.md) on how `FUN CMS` uses Docker for development.
+`Open edX` is a great tool for authoring (**Studio**), hosting and playing (**LMS**) interactive
+online courses and MOOCs.
 
+However, if you need to build a complete website with flexible content to aggregate your courses,
+in several languages and from different sources, **you will soon need a CMS**.
 
-## Installing a fresh server
+For example, the [edx.org](https://www.edx.org) website is built on `Drupal` for content pages and
+on [edX course discovery](https://github.com/edx/course-discovery) for the catalog.
 
-### Version
+At `France Université Numérique`, we wanted to build a similar portal but with `Python` and
```

Comment:
IMO "France Université Numérique" shouldn't be written has code, but rather emphasized with double quotes.

Reply:
Ok